mne/decoding/transformer.py

```diff
@@ -245,6 +245,92 @@ def inverse_transform(self, X, y=None):
         return X.reshape(-1, self.n_channels, self.n_times)
 
 
+class Vectorizer(TransformerMixin):
+    """Class to chain MNE transformer output to scikit-learn estimators.
+
+    MNE transformer have three, four dimensional output. This class converts
+    them into two dimension so as to comply with scikit-learn API.
+
+    Examples
+    -------
+    clf = make_pipeline(SpatialFilter(), XdawnTransforme(), Vectorizer(),
+                        LogisticRegression())
+    cross_val_score(clf)
+    """
+
+    def fit(self, X, y=None):
+        """Does nothing. Added for scikit-learn compatibility.
+
+        Parameters
+        ----------
+        X : numpy array, shape(n_epochs, n_chans, n_times) or
+                              (n_epochs, n_chans * n_times) or
+                              (n_epochs, n_chans, n_times, n_freqs)
```

Comment:
We need need to specify all cases, X just need to have `n_epochs` in its first dimension.

You can specify: 
`
X : array-like
    The data to fit. Can be, for example a list, or an array at least 2d. The first dimension must be of shape `n_sample`.
`